sites/fast-website/src/app/components/fast-frame/fast-frame.ts

```diff
@@ -41,12 +47,39 @@ export class FastFrame extends FASTElement {
     @observable
     public density: number = 0;
 
+    @observable
+    public borderRadius: number = 3;
+
+    @observable
+    public outlineWidth: number = 1;
+
+    @observable
+    public baseHeightMultiplier: number = 10;
+
+    @observable
+    public baseHorizontalSpacingMultiplier: number = 3;
+
+    private parsedColor = parseColorHexRGB(this.accentColor);
```

Comment:
It doesn't seem like you use this value other than to derive the accent color in `accentChangeHandler` - can this just be a local var to that fn?